class MyExcept(Exception):
    pass


class SecondEx(Exception):
    pass
